README.md

```diff
@@ -3,8 +3,8 @@
 [![codecov](https://codecov.io/gh/denoland/deno_std/branch/main/graph/badge.svg?token=w6s3ODtULz)](https://codecov.io/gh/denoland/deno_std)
 
 These modules do not have external dependencies and they are reviewed by the
-Deno core team. The intention is to have a standard set of high quality code
-that all Deno projects can use fearlessly.
+Deno core team. The intention is to have a standard set of high-quality,
+browser-compatible code that all Deno projects can use fearlessly.
```

Comment:
I'm not sure we can say this here as we haven't run test cases in browsers, also we don't have docs about how to use deno_std in browsers.

Reply:
Reverted